controllers/apiControllers.js

```diff
@@ -0,0 +1,22 @@
+function index(req, res) {
+	res.json({
+		message: 'Welcome to Marvel characters',
+		documentation_url: 'https://github.com/SherifIsmail01/express-personal-api',
+		base_url: 'localhost:3000',
+		endpoints: [
+			{
+				method: 'GET', path: '/api', description: 'send back all characters as json',
```

Comment:
I believe each endpoint should be its own object.
`{method: 'GET', path: '/api', description: 'blahblahblah'},`